### PR TITLE
gNOI-4.1 Adding Nokia deviation

### PR DESCRIPTION
**Pull request by MarcCharlebois**
Added devations to accomodate for this test passing.

"This code is a Contribution to the OpenConfig Feature Profiles project ("Work") made under the Google Software Grant and Corporate Contributor License Agreement ("CLA") and governed by the Apache License 2.0. No other rights or licenses in or to any of Nokia's intellectual property are granted for any other purpose. This code is provided on an "as is" basis without any warranties of any kind."

### CODE REVIEW

File: feature/gnoi/os/tests/osinstall/osinstall_test.go [hunk 1]
```diff
@@ -425,6 +425,10 @@ func watchStatus(t *testing.T, ic ospb.OS_InstallClient, standby bool) error {
 func TestPushAndVerifyInterfaceConfig(t *testing.T) {
 
 	dut := ondatra.DUT(t, "dut")
+	ocPortName := dut.Port(t, "port1").Name()
```
Comment (xw-g) [inline feature/gnoi/os/tests/osinstall/osinstall_test.go:428]: `ocPortName` usage is gated by `deviations.ExplicitInterfaceInDefaultVRF(dut)`, therefore it should be included in the `if` statement.

**Review by xw-g — COMMENTED** [review]
just a minor suggestion and we are good to go

File: feature/gnoi/os/tests/osinstall/osinstall_test.go (hunk 1 above)
Comment (MarcCharlebois) [reply feature/gnoi/os/tests/osinstall/osinstall_test.go]: You're absolutely correct; this is fixed in latest commit.

**Review by NishaSadhasivam-work — CHANGES_REQUESTED** [review]
With the changes test failed on Nokia vendor with the below message.

 TestOSInstall Failed with “Missing osfile or osver args” message; 
when run on image “os_version -> 0.0.0-52412”

**Comment by MarcCharlebois** [comment]
> With the changes test failed on Nokia vendor with the below message.
> 
> TestOSInstall Failed with “Missing osfile or osver args” message; when run on image “os_version -> 0.0.0-52412”

The error you've listed above suggests that you've not [defined the flags](https://github.com/openconfig/featureprofiles/blob/f59b129b977ba5440be09f5e02848d1da0e69441/feature/gnoi/os/tests/osinstall/osinstall_test.go#L53-L54) required for this test to operate successfully. However, this is not due to this PR, this would be true even in main branch. Reassigning to @NishaSadhasivam-work

**Review by NishaSadhasivam-work — APPROVED** [review]

**Pull request merged**